### ISSUE TITLE
Issues 7/16

### ISSUE THREAD

**Issue by King65868** [Saereth/inferno_issues#55]
Latex has 2 recipes in a furnace; rubber and cured rubber 
Travel staff has 2 recipes, 1 being the default one 
All pipez pipes have a default recipe + the added one 
Silver still has issues
Advanced pipe upgrade has 2 recipes, 1 being a lot more expensive than the other 
Cow stable capsule is still a 1x1x1
Marauder armor in the fletcher with bound hoglin hide has an empty output 
Know the End research references the baker
The dusts from trinkets have a default recipe 
Andesite alloy can’t be crafted without a mixer which needs andesite alloy
Red tulips cant be crafted, used in T2 tavern
Protective fabric has 2 recipes, 1 being the default one 
Occultism tallow has a recipe in the casting table with no fluid input 
Barracks tower quest wants the barracks but you only get those by building the barracks structure, although you only get 1 to start not 4 like the quests asks for
Basalt delta quest in infernal insights is 100% complete but is not actually complete 
The observation quest for a wolf doesn’t seem to work when looking at a wolf 
Blood block has 3 recipes in the casting basin, 2 that have no fluids 
Blood block has 2 recipes in the crafting table
Blood infused stone has 2 recipes, 1 is default 
You can melt blood blocks but not blood balls 
Blood balls have 2 casting table recipes, 1 is broken and is added by inferno 
Blood block and congealed blood -> blood balls has 1 default and 1 added by inferno recipes 
Crimson ladders aren’t shown in JEI

**Comment by Saereth** [comment]
dont worry about the capsules for now in this issue they''ll all be needing to get redone soon with the new buildings

feel free to fix quests you see broken otherwise I'll get to them once the recipe work is done or at least make the quest issues a separate issue please

the ladders not showing in jei I have no clue, need to harass some people, they arent set to hidden in quark's config AND im force adding them to jei but nothing... soo yeah everything else here is fixed for now though
